packages/cozy-harvest-lib/src/components/KonnectorConfiguration/ConfigurationTab/BiContractActivationWindow.jsx

```diff
@@ -23,23 +24,10 @@ const BIContractActivationWindow = ({
   onAccountDeleted
```

Comment:
Since you removed `onAccountDeleted()` call, should you remove this prop?

Reply:
This was just in the first commit. Then now, the prop is removed in the first commit and is back in the second for clarity